grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -40,6 +40,11 @@ class UrlMappings {
             }
         }
 
+        "/inventory/cycleCount/count/check" {
```

Comment:
Do we have to have a separate URL for it? Imho a simple two step wizard under `/inventory/cycleCount/count` would make more sense.

Reply:
@SebastianLib on second thought we don't even need a separate page/step for it. It is just a state with disabled editing and two buttons changed